about-page.md

```diff
@@ -0,0 +1,47 @@
+# Draft - Karrot's About page
+
+### 1st Part
+[One paragraph of text of what Karrot is about, why it exists, to whom and its vision...]
+
+Karrot exists for groups of people who want to save things from being wasted and share them instead, allowing these groups to organise efficiently, on a local and voluntary basis. Things such food, bikes, clothes or whatever is being excessively thrown away or unused in the context of a consumerist society. 
+
+Our vision, as the team of passionate volunteers developing Karrot, is to make a free and open-source software that enables a movement to grow, a movement that is based on communities and networks that save and share resources and that are socially and ecologically sustainable. Karrot has started as a foodsaving tool (hence the name) and is progressively being developed to become a platform for like-minded initiatives to connect and grow.
+
+Every group on karrot is completely independent and can have their own logo, processes and rules. Yet they are all displayed on the map so that people can find and connect to other groups in their location  and even globally.
+
+Have you already found a [group nearby you](https://karrot.world/)? Or maybe you want to [start one](https://karrot.world/#/group/create)? If you just want to try Karrot out, create an account and join the Playgroud group for testing.
```

Comment:
Small phrasing things: "If you just want to try out Karrot, then create an account and join the playground group for testing."